wasmer2/wasmer2Instance.go

```diff
@@ -103,17 +103,16 @@ func (instance *Wasmer2Instance) Clean() {
 	}
 }
 
-func (instance *Wasmer2Instance) GetPointsUsed() uint64 {
-	// return cWasmerInstanceGetPointsUsed(instance.instance)
-	return 0
+func (instance *Wasmer2Instance) SetGasLimit(gasLimit uint64) {
+	cWasmerInstanceSetGasLimit(instance.cgoInstance, gasLimit)
```

Comment:
According to `wasmer2/bridge2.go`, this function now returns a `cWasmerResultT`. Should we return an error if the result returned from Wasmer is not "success"?

Reply:
I just figured all api calls should at least return some result instead of void, as now we have that instance cast and it might fail. We can talk more about this.